### ISSUE TITLE
Resource card changes

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#365]
# Done criteria
- [ ] ?
- [ ] ?
- [ ] ?

# Designs & Docs
- [Link goes here](http://)
- [Link goes here](http://)

# BGS review criteria
[Link to review goes here](http://)
- [ ] ?
- [ ] ?
- [ ] ?

**Comment by russ-manta** [comment]
@conferences-BGS resource card image now added and can be seen on home here (1st item): http://bgs-staging.mrmdev2.co.uk/

**Comment by angievanhegan** [comment]
@conferences-BGS This one is definitely ready for you to check. Thanks

**Comment by conferences-BGS** [comment]
@BGS-Editor will test but just to confirm this is only in staging?

**Comment by angievanhegan** [comment]
@conferences-BGS @BGS-Editor yes - this will only be in staging until you put it into the 'Move to live' column. Then @russ-manta will move it to the live server. Is that ok? Let me know if I missed something.

**Comment by BGS-Editor** [comment]
@conferences-BGS @angievanhegan  resource card image now added and can be seen on home here (1st item): http://bgs-staging.mrmdev2.co.uk/

Sorry folks, I can't access back-end of the staging site in order to test out posting up an image on a resource card.

**Comment by angievanhegan** [comment]
@BGS-Editor I can see that you have an account on staging (http://bgs-staging.mrmdev2.co.uk/) with the email  editor@bgs.org.uk and you appear the have the correct privileges to edit that content.  Have you tried a password reset?

**Comment by BGS-Editor** [comment]
@angievanhegan  Thanks, Angie - but where does one log in (see screenshot)?
<img width="847" alt="screenshot 29" src="https://user-images.githubusercontent.com/43037203/48846961-50ba5200-ed98-11e8-98d7-2a40cdd876df.png">

**Comment by angievanhegan** [comment]
@BGS-Editor This this: http://bgs-staging.mrmdev2.co.uk/user
@russ-manta Where has the login button gone?

**Comment by BGS-Editor** [comment]
@angievanhegan   Sorry. I had to change my p.w on the live site. I have tried all the old and new logins on the staging site and none let me in. On the old staging site I was atkinsr and 'changeme'. My new login details are email address and BGS2018 on the live site.

**Comment by BGS-Editor** [comment]
@angievanhegan @conferences-BGS  I've tested the picture on the resource. I don't recall what the argument was about pictures on resources. I can see the pic if it appears in the featured content at the top of the home page. I can't see the pic on the 'Resources' catalogue page; or on the homepage under the 'Resources' tab. That is not an issue for me. What about you G?

**Comment by conferences-BGS** [comment]
@russ-manta http://bgs-staging.mrmdev2.co.uk/resources no images appear on this page. Was this not the case

**Comment by BGS-Editor** [comment]
@russ-manta @conferences-BGS Yes, that was it and whereas it is not an issue for me, it will be an issue for Colin as I think it was he who wanted it.

**Comment by angievanhegan** [comment]
@conferences-BGS The top of each card shows the requirements we took from Steph. The main purpose for adding these card images is so that you can show the resources at the top of the home page.  They could theoretically also appear on the reources page or the 'Latest' section of the homepage. But, a decision was taken earlier on for these parts of the site not to have images. It might be strange for some items to have images and others not.  Hope that's helpful.

**Comment by BGS-Editor** [comment]
@conferences-BGS @angievanhegan  - I'm okay with that. I think I'm going to leave this in G's hands, to negotiate with Colin.

**Comment by conferences-BGS** [comment]
@angievanhegan Was the decision made by Colin or Steph? If Steph, the latest view I think that images need to appear on these cards on this http://bgs-staging.mrmdev2.co.uk/resources

**Comment by BGS-Editor** [comment]
@angievanhegan @conferences-BGS Okay. I don't think we need to worry about some pieces having pics and some not. We would treat non-pic pieces the same way that we treat policy and media page, where we just put holding images onto non-pic pieces.

**Comment by angievanhegan** [comment]
@conferences-BGS I really appreciate that it would be lovely to have images on these resources cards, particularly if you had images for all of them, but are you sure you want to run the risk of having a page full of the 'blank' images? The main priority that Colin spoke of at the start was the make the content easier to find. Looking at the two examples side by side below, I think that resource titles, without the noise of the blank image, is much easier for the user to scan to pick out what it is they're searching for. It's also worth taking into account that even the images you do have, whilst they may be very attractive, colourful and carefully selected, they might not necessarily make it easier for people to find the content they're searching for.  A lot of BGS time and effort went in to making the resources pages work as well as they do, I think it's worth being sure that any additional changes are really going to improve the experience and search-ability for users.

**Comment by BGS-Editor** [comment]
@conferences-BGS @angievanhegan . I must say, I agree that the blank blocks looks clearer, although the blocks with the holding images would also have a white block below the pics like this page: https://www.bgs.org.uk/policy-and-media. I'm going to leave it to G and Colin. I would select the pictureless blocks, but I'm not the one who is going to work with it in the future.

**Comment by angievanhegan** [comment]
@BGS-Editor @conferences-BGS I thought it was worth clarifying, our recommendation is that no images is easier for users, then either some images + some blank spaces or some images + some coloured holding block.

**Comment by conferences-BGS** [comment]
> @BGS-Editor @conferences-BGS I thought it was worth clarifying, our recommendation is that no images is easier for users, then either some images + some blank spaces or some images + some coloured holding block.

Colin concurs with MRMs suggestions - no resource card images to be added . Leave as is

**Comment by mrm-git** [comment]
➤ Angie Vanhegan commented:

@russurwin This one can be moved to live.
